more/free-programming-interactive-tutorials-en.md

```diff
@@ -138,6 +139,7 @@
 
 ### Go
 
+* [Go at Codecademy](https://www.codecademy.com/learn/learn-go)
```

Comment:
Right title, instructor attributions, platform and signup notes are always the best contribution. Follow format

```suggestion
* [Learn Go](https://www.codecademy.com/learn/learn-go) - Google, Robert Griesemer, Rob Pike, and Ken Thompson (Codecademy) *(account required)*
```